recipe/meta.yaml

```diff
@@ -357,7 +357,9 @@ outputs:
         - python
         - numpy >=1.26
         - cupy >=13.0
-        - {{ pin_compatible('nvmath-python', max_pin='x.x') }}
+        - cuda-bindings >=12.9.1,<13.0.0  # [(cuda_compiler_version or "").startswith("12")]
+        - cuda-bindings >=13.0.1,<14.0.0  # [(cuda_compiler_version or "").startswith("13")]
+        - {{ pin_compatible('nvmath-python=*=core*', max_pin='x.x') }}
```

Comment:
`pin_compatible` is when a dependency is used at build time and there is associated run-time constraint that we want to impose. `nvmath-python` is not a build-time dep to the best of my knowledge. Just treat it normally like numpy/cupy.

Reply:
fixed in 4754e6b83069da4e0907970af83e5f7883922a1f